site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -183,21 +183,38 @@ please [file an issue](https://github.com/googleforgames/agones/issues).
 #### SetLabel(key, value)
 
 This will set a [Label](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) value on the backing `GameServer`
-record that is stored in Kubernetes. To maintain isolation, the `key` value is automatically prefixed with "agones.dev/sdk-".
+record that is stored in Kubernetes. 
+
+To maintain isolation, the `key` value is automatically prefixed with value **"agones.dev/sdk-"**. This is done for two 
+main reasons:
+*  The prefix allows the developer to always know if they are accessing or reading a value that could have come, or 
+   may be changed by the client SDK. Much like `private` vs `public` scope in a programming language - The Agones SDK only 
+   gives you access to part of the set of labels and annotations that exist on a GameServer.
+*  The prefix allows for a smaller attack surface if the GameServer container gets compromised. Since the 
+   game container is generally externally exposed, and the Agones project doesn't control the binary that is 
+   run within it - limiting exposure if something goes horribly wrong, and it becomes compromised is worth the extra 
```

Comment:
nit: remove , after wrong

Reply:
grammar check on IntelliJ tells me to comma after wrong 😄

Reply:
Maybe that's regex matching because it is following by and? The way I read it "something goes horribly wrong and it becomes compromised" is a single phrase, not two phrases that need to be separated by a comma. But maybe my pattern matching is off....

Reply:
removed the "horribly wrong" bit - also reads cleaner and less emotionally charged language 😄